### DISCUSSION TITLE
[FF7] Save/PHS anywhere seems to be not working on last few canary builds

### DISCUSSION THREAD

**Discussion by Gazlighter** [julianxhokaxhiu/FFNx#399] [Q&A] [answered]
Pretty self explanatory. After I updated to the latest canary, this feature stopped working. Couldn't find anything in FFNX.toml that referenced the feature.

**Comment by julianxhokaxhiu** [comment, marked as answer]
It's not a bug, it was intended as it bring more issues than gains. Consider not using that feature in general, and if you do, get ready for massive issues when loading the game on top of those save files.

**Reply by Gazlighter** [reply]
No problems. I wasn't actually using it for saving, just for the PHS so have never had any problems with it. It's great to just swap out party members whenever you feel like it. Is there a way to re-enable it? Thanks for everything that you do. Brilliant work.

**Reply by Gazlighter** [reply]
I thought it was unintended due to it still being listed as a feature on the main page.

**Reply by julianxhokaxhiu** [reply]
Thank you. I'm glad you enjoy using FFNx. Regarding:

> I thought it was unintended due to it still being listed as a feature on the main page.

I am sorry, I forgot to update that part. I will now. Thanks!

**Reply by Gazlighter** [reply]
All good. So there's no way to manually re-enable the feature so I can use PHS anywhere?

**Reply by julianxhokaxhiu** [reply]
No sorry, not officially via FFNx.

**Reply by Gazlighter** [reply]
That's ok. Can live without it. Thanks anyway.
